repology/parsers/parsers/sisyphus.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2021 Danil Shein <dshein@altlinux.org>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Iterable
+
+from repology.packagemaker import NameType, LinkType, PackageFactory, PackageMaker
+from repology.parsers import Parser
+from repology.parsers.json import iter_json_list
+
+
+class SisyphusJsonParser(Parser):
+    def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMaker]:
+        for packagedata in iter_json_list(path, ('packages', None)):
+            with factory.begin() as pkg:
+                pkg.add_name(packagedata['name'], NameType.GENERIC_GEN_NAME)
```

Comment:
NameType.SRCRPM_NAME is suitable here.

Reply:
In fact "name" field in our JSON contains source package name excluding epoch, version, release, architecture etc.